devs-who-run-api/appsettings.json

```diff
@@ -5,16 +5,28 @@
       "Microsoft.AspNetCore": "Warning"
     }
   },
+  "AllowedHosts": "*",
   "Database": {
-    "ConnectionString": "Server=localhost;Port=5432;Database=devswhorun;User Id=postgres;Password=admin;",
+    "ConnectionString": "Host=localhost;Port=5432;Database=devswhorun;Username=postgres;Password=postgres",
```

Comment:
any reason this is changed?

Reply:
Nice catch, yes I need to remove this one as this is my machine's configuration